pennylane/transforms/metric_tensor.py

```diff
@@ -292,12 +325,226 @@ def processing_fn(probs):
 
         for prob, obs, coeffs in zip(probs, obs_list, coeffs_list):
             # calculate the covariance matrix of this layer
-            scale = qml.math.convert_like(np.outer(coeffs, coeffs), prob)
+            scale = qml.math.convert_like(qml.math.outer(coeffs, coeffs), prob)

```

Comment:
I believe because we tests differentiability based on single indexed entries of the metric tensor... and users might not be differentiating the metric tensor very much :D

Reply:
😆

Reply:
If I cast my mind back, only one person has ever requested a differentiable metric tensor (I think it was @johannesjmeyer)

Reply:
More accurately, I think it was a differentiable covariance matrix (which is what the block-diag approx essentially is)

Reply:
I did request this and this is a really cool feature! :) Also nice to finally have the full metric tensor available, this will make my future numerics much easier.